wallet/dapps-communication.md

```diff
@@ -463,6 +705,81 @@ items:
 </tbody>
 </table>
 
+#### Generating New Accounts
+
+This method will generate a new random account upon request with a specified name defined by the user. The account shall be added to the wallet automatically.
+
+<table>
+<thead>
+<tr>
+<th>Direction</th>
+<th>Type</th>
+<th>Format</th>
+<th>Example</th>
+</tr>
+</thead>
+<tbody>
+
+<tr>
+<td>Webpage to extension</td>
+<td><code>account.generateNewAccount</code></td>
+<td>
+
+```yaml
+type: object
+properties:
+  name:
```

Comment:
we named it nickname on thyra side, do we need to be consistent?

Reply:
@gregLibert So far we have been using name in the `wallet-provider`. Do we stick with it or do a general renaming ?

Reply:
No, name is clearer. We should live it like that